app/controllers/admin/shelters_controller.rb

```diff
@@ -0,0 +1,6 @@
+class Admin::SheltersController < ApplicationController
+  def index
+    @shelters = Shelter.order_by_reverse_alphabetical
+    @pending_apps = Shelter.pending_apps.alphabetical_order
```

Comment:
Nice work encapsulating this logic at the model level rather than doing it all here!